blocks/header-nav-chain-block/themes/news.json

```diff
@@ -131,6 +131,15 @@
 			"desktop": {}
 		}
 	},
+	"header-nav-chain-logo-right": {
+		"styles": {
+			"default": {
+				"margin-left": "auto",
+				"margin-right": 0
```

Comment:
Since we don't have to worry about any component conflicts, lets use the correct logical properties here, `margin-inline-start: auto` and `margin-inline-end: 0`.